sdks/python/apache_beam/transforms/environments.py

```diff
@@ -252,6 +254,14 @@ def from_runner_api_parameter(payload, capabilities, artifacts, context):
   @classmethod
   def from_options(cls, options):
     # type: (PipelineOptions) -> DockerEnvironment
+    if options.view_as(DebugOptions).lookup_experiment(
+        'prebuild_sdk_container'):
+      prebuilt_container_image = SdkContainerBuilder.build_container_imge(
```

Comment:
I see this being called both here and in dataflow_runner.py. Are we invoking it twice on Dataflow?

Reply:
Changed to invoking from_options in dataflow runner.